dashboard/src/shared/InstalledPackage.ts

```diff
@@ -105,3 +115,21 @@ export class InstalledPackage {
     } as DeleteInstalledPackageRequest);
   }
 }
+
+// Helpers to be able to call with a retry backing off exponentially
+// with 500ms, 1000ms, 2000ms, 4000ms etc.
+const wait = (ms: number) => new Promise(res => setTimeout(res, ms));
+
+const callWithRetry = async (fn: any, initialWait: number, depth = 0): Promise<any> => {
```

Comment:
What if we just use a library instead for also [adding jitter](https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/) to this exp backoff, like https://github.com/lifeomic/attempt ?

Reply:
@antgamdia thanks for the article about jitter, superinteresting!
That library looks easy to implement 😄

Reply:
Why would we need jitter here? It's guaranteed to be a single client fetching the resource refs after creating the package... unless we coordinate a DDOS on a Kubeapps installation :P. (ie. we don't have the situation described in the article that leads to adding jitter to an exponential backoff, nor could we, unless I'm missing something?)